File: ABC/Python3/ABC129/B.py
N = int(input())
W = list(map(int,input().split()))

ans = 1e9
for i in range(N):
  l = sum(W[:i+1])
  r = sum(W[i+1:])
  ans = min(ans, abs(l-r))
print(ans)

File: ABC/ABC129/B.cpp
#include <iostream>
#include <vector>

#define rep(i,n) for(int i=0;i<n;++i)

using namespace std;

int main(){
  int N;
  cin >> N;
  vector<int> W(N);
  rep(i,N) cin >> W[i];
  int ans = (int)1e9+7;
  rep(T,N){
    int left = 0, right = 0;
    for(int i = 0; i<T; ++i){
      left += W[i];
    }
    for (int i = T; i<N; ++i){
      right += W[i];
    }
    ans = min(ans, abs(right-left));
  }
  cout << ans << endl;
}
